src/Serializer/EnvelopItems/MetricsItem.php

```diff
@@ -37,6 +37,12 @@ public static function toEnvelopeItem(Event $event): string
         $metricMetaPayload = [];
 
         foreach ($metrics as $metric) {
+            /**
+             * In case of us adding support for setting a namespace for metrics,
```

Comment:
Tiny nitpick: Not necessarily just setting namespaces, but simply the SDK emitting metrics from other namespaces. That could also be built-in functionality without an explicit user-facing API, such as client reports.